packages/frontend/src/utils/kpi.ts

```diff
@@ -18,17 +18,13 @@ export function getDistributableRewardsPercentage(
     lowerUsdTarget: number,
     upperUsdTarget: number,
     minimumPayoutPercentage?: number,
-    kpiMeasurement?: number,
 ) {
     let reachedPercentage = getReachedGoalPercentage(
```

Comment:
This can be a `const` now.

Reply:
Addressed in e7ce1ee8efd018ba06721a2550d544a1b694dfea